src/utils/lib.ml

```diff
@@ -991,6 +991,14 @@ let rec find_file filename = function
     else find_file filename include_dirs
 
 
+(* Gets the relative path of the file name given a file path *)
+let get_relative_path fpath =
+  if Filename.is_relative fpath
+  then fpath
+  else let pwd = Sys.getcwd () in
+       String.sub fpath (1 + String.length pwd) (String.length fpath - String.length pwd - 1) 
```

Comment:
This line is problematic. Need more information as to what we should be displaying.